src/styles/Input.scss

```diff
@@ -39,19 +33,75 @@
             outline-style: solid;
         }
 
+        &:disabled{
+            border-radius: 0rem !important;
+            border: 1px $LIGHT-GREY;
+            border-style: solid;
+            background: $WHITE;
+        }
+    }
+
+    ::placeholder{
+        font-weight: 400;
+        text-transform: none;
+        font-size: 0.7rem;
+        color: $LIGHT-GREY;
+    }
+
+    .clr-subtext{
+        display: block;
+        font-size: 0.6rem;
+        line-height: 2rem;
+        margin-top: .3rem;
+        color: $DARK-GREY-HELPERTEXT;
+        font-weight: 400;
+        margin-top: 1.5rem;
+        margin-left: -16.4rem;
     }
 
+    &.clr-error{
+        .clr-subtext{
+            color: $ERROR;
+            line-height: 20px;
+            margin-top: 2rem;
+            margin-left: -17.5rem;
+        }
+
+        .clr-validate-icon {
+            color: $ERROR;
+            margin-top: 0.4rem;
+        }
+    
+        input{
+            border-color: $ERROR !important;
+        }
+    }
+
+    &.clr-form-control-disabled{
+        label{
+            color: $LIGHT-GREY;
+        }
+    }
+
+    label {
+        width: max-content;
+        margin-right: 1.5rem;
+        font-weight: 400;
+        letter-spacing: initial;
+        align-content: center;
+        margin-top: 0.35rem;
+        margin-left: 1.5rem;
+        font-size: 0.7rem;
+        color: $BLACK;
+    }
 }
 
-label{
-    width: max-content;
-    margin-right: 1.5rem;
+.input-header{
     font-weight: bold;
-    letter-spacing: initial;
-    align-content: center;
-    margin-top: 0.35rem;
+    font-size: 0.7rem;
+    line-height: 20px;
+    color: $BLACK;
     margin-left: 1.5rem;
-    size: 14px;
 }
 
 .topLabel{
```

Comment:
Do we need `topLabel` ? or can be removed ?

Reply:
We can remove it. Removed it.